packages/flutter_tools/lib/src/doctor.dart

```diff
@@ -915,19 +915,31 @@ class IntelliJValidatorOnMac extends IntelliJValidator {
     }
 
     final List<String> split = version.split('.');
-
     if (split.length < 2) {
       return null;
     }
-
     final String major = split[0];
     final String minor = split[1];
-    _pluginsPath = globals.fs.path.join(
-      globals.fsUtils.homeDirPath,
+
+    final String homeDirPath = globals.fsUtils.homeDirPath;
+    String pluginsPath = globals.fs.path.join(
+      homeDirPath,
       'Library',
       'Application Support',
+      'JetBrains',
       '$id$major.$minor',
+      'plugins',
     );
+    if (!globals.fs.isDirectorySync(pluginsPath)) {
```

Comment:
could we add a comment here to explain why we have two paths?

Reply:
👍 I had suggested a comment in https://github.com/flutter/flutter/pull/57963#discussion_r430725149

Reply:
Code updated with comment 👍